python2verilog/ir/graph.py

```diff
@@ -105,6 +105,12 @@ def child(self) -> Element:
     def child(self, other: Element):
         self._child = typed_strict(other, Element)
 
+    def has_child(self) -> bool:
+        """
+        Returns True if has child
```

Comment:
If what has child?

This is also a graph, so what's your definition of a child?

Reply:
> If what has child?

Returns `True`

> This is also a graph, so what's your definition of a child?

This is a member of `BasicElement`, which is a subclass of `Element` that has only one child. E.g., it is the superclass for edges and assign nodes, but not a superclass for if-else nodes (as they have two children)